app/src/main/res/layout/fragment_adminpanel_dailymarket.xml

```diff
@@ -0,0 +1,307 @@
+                app:layout_constraintStart_toStartOf="@+id/rewardEditText"
+                app:layout_constraintTop_toBottomOf="@+id/rewardEditText" />
+
+            <View
+                android:id="@+id/view2"
+                android:layout_width="match_parent"
+                android:layout_height="2dp"
+                android:layout_margin="5dp"
+                android:background="@color/black"
+                app:layout_constraintTop_toBottomOf="@+id/addDailyChallengeButton"
+                tools:layout_editor_absoluteX="8dp" />
+
+
+            <TextView
+                android:id="@+id/updatedailyChallengeTextView"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:backgroundTint="@color/neutral_font_color"
+                android:hint="Update DailyChallenge"
+                android:inputType="number"
+                android:textColor="@color/neutral_font_color"
+                android:textColorHint="@color/neutral_font_color"
+                app:layout_constraintBottom_toBottomOf="@+id/closeDailyChallengeButton"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="@+id/openDailyChallengeButton" />
+
+            <Button
+                android:id="@+id/openDailyChallengeButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_margin="5dp"
+                android:layout_marginTop="8dp"
+                android:background="@drawable/black_white_rounded_button"
+                android:padding="10dp"
+                android:text="Open DailyChallenge"
+                android:textColor="@color/neon_green"
+                app:layout_constraintEnd_toEndOf="@+id/addDailyChallengeButton"
+                app:layout_constraintHorizontal_bias="0.4"
+                app:layout_constraintStart_toStartOf="@+id/addDailyChallengeButton"
+                app:layout_constraintTop_toBottomOf="@+id/view2" />
+
+            <Button
+                android:id="@+id/closeDailyChallengeButton"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_margin="5dp"
+                android:layout_marginTop="4dp"
+                android:background="@drawable/black_white_rounded_button"
+                android:padding="10dp"
+                android:text="Close DailyChallenge"
+                android:textColor="@color/neon_yellow"
+                app:layout_constraintEnd_toEndOf="@+id/openDailyChallengeButton"
+                app:layout_constraintHorizontal_bias="0.166"
+                app:layout_constraintStart_toStartOf="@+id/openDailyChallengeButton"
+                app:layout_constraintTop_toBottomOf="@+id/openDailyChallengeButton" />
+            <View
+                android:id="@+id/view4"
```

Comment:
bad way for naming a layout !

This may looks fine for now but not a good practice to handle OpenSource projects !

Please rename all views to some meaning names :)